pstore/kafka/kafka.go

```diff
@@ -5,9 +5,9 @@ import (
 	"encoding/json"
 	"fmt"
 	"github.com/Symantec/scotty/pstore"
+	"github.com/Symantec/scotty/pstore/lmm"
```

Comment:
I will move these functions to the Kafka pkg. at lunch and running errands
now

On Monday, May 9, 2016, rgooch <notifications@github.com> wrote:

> In pstore/kafka/kafka.go
> <https://github.com/Symantec/scotty/pull/67#discussion_r62539015>:
>
> > @@ -5,9 +5,9 @@ import (
> >  	"encoding/json"
> >  	"fmt"
> >  	"github.com/Symantec/scotty/pstore"
> > +	"github.com/Symantec/scotty/pstore/lmm"
>
> Why is this called lmm rather than kafka? The former is an internal,
> non-generic name.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Symantec/scotty/pull/67/files/359e0a199f4b400d1c1d5062acbb77849edb4c7a#r62539015>
>